src/System.Private.CoreLib/src/System/Diagnostics/Eventing/RuntimeEventSourceHelper.OSX.cs

```diff
@@ -0,0 +1,79 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Runtime.InteropServices;
+
+namespace System.Diagnostics.Tracing
+{
+    internal sealed class RuntimeEventSourceHelper
+    {
+        // from sys\resource.h
+        [StructLayout(LayoutKind.Sequential)]
+        private unsafe struct rusage_info_v3
+        {
+            internal fixed byte     ri_uuid[16];
```

Comment:
The struct is `private` inside of an `internal` class. The fields can be marked `public`.